azure-devops/cmake-configure-build.yml

```diff
@@ -49,3 +48,27 @@ steps:
   displayName: 'Build the STL'
   timeoutInMinutes: 10
   env: { TMP: $(tmpDir), TEMP: $(tmpDir) }
+- script: |
+    if exist "$(${{ parameters.benchmarkBuildOutputLocationVar }})" (
+      rmdir /S /Q "$(${{ parameters.benchmarkBuildOutputLocationVar }})"
+    )
+    call "%ProgramFiles%\Microsoft Visual Studio\2022\Preview\Common7\Tools\VsDevCmd.bat" ^
+    -host_arch=${{ parameters.hostArch }} -arch=${{ parameters.targetArch }} -no_logo
+    cmake ${{ parameters.cmakeAdditionalFlags}} -G Ninja ^
+    -DCMAKE_CXX_COMPILER=cl ^
+    -DCMAKE_BUILD_TYPE=Release ^
+    -DLIT_FLAGS=$(litFlags) ^
+    -DSTL_USE_ANALYZE=ON ^
+    -S $(Build.SourcesDirectory) -B $(${{ parameters.benchmarkBuildOutputLocationVar }})
+  displayName: 'Configure the benchmarks'
+  timeoutInMinutes: 2
+  env: { TMP: $(tmpDir), TEMP: $(tmpDir) }
+  condition: ne('${{ parameters.benchmarkBuildOutputLocationVar }}', '')
+- script: |
+    call "%ProgramFiles%\Microsoft Visual Studio\2022\Preview\Common7\Tools\VsDevCmd.bat" ^
+    -host_arch=${{ parameters.hostArch }} -arch=${{ parameters.targetArch }} -no_logo
+    cmake --build $(${{ parameters.benchmarkBuildOutputLocationVar }})
+  displayName: 'Build the benchmarks'
+  timeoutInMinutes: 10
+  env: { TMP: $(tmpDir), TEMP: $(tmpDir) }
+  condition: ne('${{ parameters.benchmarkBuildOutputLocationVar }}', '')

```

Comment:
This isn't configuring and building the benchmarks - it's configuring and building the STL again. :scream_cat: This is visible in the Azure Pipelines logs, where "Build the benchmarks" took 1m 21s, and built the STL's 1014 files.

Comparing the build/configure scripts for the STL above to the scripts for the benchmarks below reveals what happened. As desired, there are changes for the benchmarks' different output location (and `condition`s to skip this if that location has been overridden to be empty, as ARM does), and the display names are updated.

However, what's missing are the steps described in the README: `-S` needs to be given the `benchmarks` subdirectory, and `-DSTL_BINARY_DIR=meow` needs to point to the freshly built STL.